packages/web/src/app/onboard/components/memberApprovalRequiredToggle.tsx

```
@@ -0,0 +1,125 @@
+import { useState, useEffect } from "react"
+import { Switch } from "@/components/ui/switch"
+import { Loader2 } from "lucide-react"
+import { setMemberApprovalRequired, getMemberApprovalRequired } from "@/actions"
+import { SINGLE_TENANT_ORG_DOMAIN } from "@/lib/constants"
+import { isServiceError } from "@/lib/utils"
+import { useToast } from "@/components/hooks/use-toast"
+import { InviteLinkToggle } from "@/app/components/inviteLinkToggle"
+
+interface MemberApprovalRequiredToggleProps {}
+
+export function MemberApprovalRequiredToggle({}: MemberApprovalRequiredToggleProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)